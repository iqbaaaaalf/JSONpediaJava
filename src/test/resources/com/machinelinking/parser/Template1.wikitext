{{Infobox
|bodyclass  = vcard
|aboveclass = fn
|above      = {{#if:{{{name|}}}|{{{name}}}|{{PAGENAME}}}}
|abovestyle = background:transparent; text-align:center;

|image      = {{#invoke:InfoboxImage|InfoboxImage|image={{{image|}}}|size={{{image_size|{{{imagesize|}}}}}}|sizedefault=frameless|alt={{{alt|}}}|suppressplaceholder=yes}}
|imagestyle = padding-bottom:0.5em;
|caption    = {{{caption|}}}
|labelstyle = background:transparent; line-height:1.1em;
|datastyle  = line-height:1.3em; vertical-align:middle;

| label1     = Native name
| data1      = {{#if:{{{native_name|}}}|<span class="nickname" {{#if:{{{native_name_lang|}}}|lang="{{{native_name_lang}}}" xml:lang="{{{native_name_lang}}}"}}>{{{native_name}}}</span> }}

|label10  = Born
|data10   = {{br separated entries|{{{birth_name|}}}|{{{birth_date|}}}|{{{birth_place|}}}}}

|label11  = Died
|data11   = {{br separated entries|{{{death_date|}}}|{{{death_place|}}}|{{{death_cause|}}}}}

| label12     = Resting place
| class12     = label
| data12      = {{br separated entries|{{{resting place|{{{resting_place|{{{restingplace|}}}}}}}}}|{{{resting place coordinates|{{{resting_place_coordinates|{{{restingplacecoordinates|}}}}}}}}}}}

| label13    = Other names
| class13    = nickname
| data13     = {{{other_names|}}}

|label14  = Residence
|data14   = {{{residence|}}}
|label15  = Citizenship
|data15   = {{{citizenship|}}}
|label16  = Nationality
|data16   = {{{nationality|}}}

|label17  = Fields
|class17  = category
|data17   = {{{field|{{{fields|}}}}}}

|label18  = Institutions
|data18   = {{{workplaces|}}}<!--
        -->{{{work_institutions|}}}{{{work_institution|}}}

|label19  = [[Alma mater]]
|data19   = {{{alma_mater|}}}
|label30 = [[Thesis]]
|data30  = {{#if:{{{thesis_title|}}}|{{#if:{{{thesis_url|}}}|[{{{thesis_url}}} ''{{{thesis_title}}}'']|''{{{thesis_title}}}''}}}} {{#if:{{{thesis_year|}}}|({{{thesis_year}}})}}
|label31 = [[Doctoral advisor]]
|data31  = {{{doctoral_advisor|}}}

|label32 = {{#if:{{{doctoral_advisor|}}}
             | Other&nbsp;academic advisors
             | Academic advisors }}
|data32  = {{{academic_advisors|}}}

|label33 = Doctoral students
|data33  = {{{doctoral_students|}}}

|label34 = {{#if:{{{doctoral_students|}}}
             | Other&nbsp;notable students
             | Notable students }}
|data34  = {{{notable_students|}}}

|label35 = Known&nbsp;for
|data35  = {{{known_for|}}}
|label36 = Influences
|data36  = {{{influences|}}}
|label37 = Influenced
|data37  = {{{influenced|}}}
|label38 = Notable awards
|data38  = {{{awards|}}}{{{prizes|}}}
|label39 = [[Author citation (botany)|Author abbreviation (botany)]]
|data39  = {{{author_abbreviation_bot|{{{author_abbrev_bot|}}}}}}
|label40 = [[Author citation (zoology)|Author abbreviation (zoology)]]
|data40  = {{{author_abbreviation_zoo|{{{author_abbrev_zoo|}}}}}}
|label51 = Spouse
|data51  = {{{spouse|}}}
|label52 = Partner(s)
|data52  = {{{partner|}}}
|label53 = Children
|data53  = {{{children|}}}
|data54  = {{#if:{{{signature|}}}
             | '''Signature'''<br/><!--
               -->[[File:{{{signature}}}|128px<!--
                 -->|alt={{{signature_alt|}}}<!--
                 -->|{{{name|{{PAGENAME}}}}}'s signature]] }}

|header65 = {{#if:{{{website|{{{homepage|{{{URL|}}}}}}}}}|Website}}
|data66   = {{{website|{{{homepage|{{{URL|}}}}}}}}}

|data67  = {{#if:{{{footnotes|}}}
             | <div style="padding-top:0.25em; text-align:left; font-style:90%;"><!--
                    -->'''Notes'''<br/>{{{footnotes}}}</div> }}
}}<!--

-->{{#if:{{{author_abbreviation_zoo|{{{author_abbrev_zoo|}}}}}}
     | [[Category:Zoologists with author abbreviations<!--
         -->|{{{author_abbreviation_zoo|{{{author_abbrev_zoo}}}}}}]]<!--
-->}}{{#if:{{{pronunciation|}}}|[[Category:Biography template using pronunciation]]}}<noinclude>
{{documentation}}
<!---Please add metadata to the  section at the bottom of the /doc subpage--->
</noinclude>
