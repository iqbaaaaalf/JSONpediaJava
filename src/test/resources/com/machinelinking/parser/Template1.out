Begin Document
Begin Template: Infobox

k: bodyclass  
Text: 'vcard
'
k: aboveclass 
Text: 'fn
'
k: above      
Begin Template: #if:<name:>
k: null
var: name [null]
k: null
Begin Template: PAGENAME
End Template: PAGENAME
End Template: #if:<name:>
Text: '
'
k: abovestyle 
Text: 'background:transparent; text-align:center;

'
k: image      
Begin Template: #invoke:InfoboxImage
k: null
Text: 'InfoboxImage'
k: image
var: image [const: []]
k: size
var: image_size [var: imagesize [const: []]]
k: sizedefault
Text: 'frameless'
k: alt
var: alt [const: []]
k: suppressplaceholder
Text: 'yes'
End Template: #invoke:InfoboxImage
Text: '
'
k: imagestyle 
Text: 'padding-bottom:0.5em;
'
k: caption    
var: caption [const: []]
Text: '
'
k: labelstyle 
Text: 'background:transparent; line-height:1.1em;
'
k: datastyle  
Text: 'line-height:1.3em; vertical-align:middle;

'
k: label1     
Text: 'Native name
'
k: data1      
Begin Template: #if:<native_name:>
Open Tag: span attributes: [class : 'nickname', {{#if:{{{native_name_lang|}}}|lang : '{{{native_name_lang}}}', xml:lang : '{{{native_name_lang}}}', null : '}}']
k: null
var: native_name [null]
Close Tag: span
Text: ' '
End Template: #if:<native_name:>
Text: '

'
k: label10  
Text: 'Born
'
k: data10   
Begin Template: br separated entries
k: null
var: birth_name [const: []]
k: null
var: birth_date [const: []]
k: null
var: birth_place [const: []]
End Template: br separated entries
Text: '

'
k: label11  
Text: 'Died
'
k: data11   
Begin Template: br separated entries
k: null
var: death_date [const: []]
k: null
var: death_place [const: []]
k: null
var: death_cause [const: []]
End Template: br separated entries
Text: '

'
k: label12     
Text: 'Resting place
'
k: class12     
Text: 'label
'
k: data12      
Begin Template: br separated entries
k: null
var: resting place [var: resting_place [var: restingplace [const: []]]]
k: null
var: resting place coordinates [var: resting_place_coordinates [var: restingplacecoordinates [const: []]]]
End Template: br separated entries
Text: '

'
k: label13    
Text: 'Other names
'
k: class13    
Text: 'nickname
'
k: data13     
var: other_names [const: []]
Text: '

'
k: label14  
Text: 'Residence
'
k: data14   
var: residence [const: []]
Text: '
'
k: label15  
Text: 'Citizenship
'
k: data15   
var: citizenship [const: []]
Text: '
'
k: label16  
Text: 'Nationality
'
k: data16   
var: nationality [const: []]
Text: '

'
k: label17  
Text: 'Fields
'
k: class17  
Text: 'category
'
k: data17   
var: field [var: fields [const: []]]
Text: '

'
k: label18  
Text: 'Institutions
'
k: data18   
var: workplaces [const: []]
Comment Tag: 
        
var: work_institutions [const: []]
var: work_institution [const: []]
Text: '

'
k: label19  
Begin Reference: Alma mater
End Reference: Alma mater
Text: '
'
k: data19   
var: alma_mater [const: []]
Text: '
'
k: label30 
Begin Reference: Thesis
End Reference: Thesis
Text: '
'
k: data30  
Begin Template: #if:<thesis_title:>
k: null
Begin Template: #if:<thesis_url:>
k: null
Begin Link: null
var: thesis_url [null]
k: null
Text: ' '''
var: thesis_title [null]
Text: ''''
End Link: null
k: null
Text: ''''
var: thesis_title [null]
Text: ''''
End Template: #if:<thesis_url:>
End Template: #if:<thesis_title:>
Text: ' '
Begin Template: #if:<thesis_year:>
k: null
Text: '('
var: thesis_year [null]
Text: ')'
End Template: #if:<thesis_year:>
Text: '
'
k: label31 
Begin Reference: Doctoral advisor
End Reference: Doctoral advisor
Text: '
'
k: data31  
var: doctoral_advisor [const: []]
Text: '

'
k: label32 
Begin Template: #if:<doctoral_advisor:>
             
k: null
Text: 'Other&nbsp;academic advisors
             '
k: null
Text: 'Academic advisors '
End Template: #if:<doctoral_advisor:>
             
Text: '
'
k: data32  
var: academic_advisors [const: []]
Text: '

'
k: label33 
Text: 'Doctoral students
'
k: data33  
var: doctoral_students [const: []]
Text: '

'
k: label34 
Begin Template: #if:<doctoral_students:>
             
k: null
Text: 'Other&nbsp;notable students
             '
k: null
Text: 'Notable students '
End Template: #if:<doctoral_students:>
             
Text: '
'
k: data34  
var: notable_students [const: []]
Text: '

'
k: label35 
Text: 'Known&nbsp;for
'
k: data35  
var: known_for [const: []]
Text: '
'
k: label36 
Text: 'Influences
'
k: data36  
var: influences [const: []]
Text: '
'
k: label37 
Text: 'Influenced
'
k: data37  
var: influenced [const: []]
Text: '
'
k: label38 
Text: 'Notable awards
'
k: data38  
var: awards [const: []]
var: prizes [const: []]
Text: '
'
k: label39 
Begin Reference: Author citation (botany)
k: null
Text: 'Author abbreviation (botany)'
End Reference: Author citation (botany)
Text: '
'
k: data39  
var: author_abbreviation_bot [var: author_abbrev_bot [const: []]]
Text: '
'
k: label40 
Begin Reference: Author citation (zoology)
k: null
Text: 'Author abbreviation (zoology)'
End Reference: Author citation (zoology)
Text: '
'
k: data40  
var: author_abbreviation_zoo [var: author_abbrev_zoo [const: []]]
Text: '
'
k: label51 
Text: 'Spouse
'
k: data51  
var: spouse [const: []]
Text: '
'
k: label52 
Text: 'Partner(s)
'
k: data52  
var: partner [const: []]
Text: '
'
k: label53 
Text: 'Children
'
k: data53  
var: children [const: []]
Text: '
'
k: data54  
Begin Template: #if:<signature:>
             
k: null
Text: ''''Signature''''
Inline Tag: br attributes: []
Comment Tag: 
               
Begin Reference: File:{
k: null
Text: '128px'
Comment Tag: 
                 
k: alt
var: signature_alt [const: []]
Comment Tag: 
                 
k: null
Begin Template: PAGENAME
End Template: PAGENAME
var: name [null]
Text: ''s signature'
End Reference: File:{
Text: ' '
End Template: #if:<signature:>
             
Text: '

'
k: header65 
Begin Template: #if:<website:<homepage:<URL:>>>
k: null
Text: 'Website'
End Template: #if:<website:<homepage:<URL:>>>
Text: '
'
k: data66   
var: website [var: homepage [var: URL [const: []]]]
Text: '

'
k: data67  
Begin Template: #if:<footnotes:>
             
Open Tag: div attributes: [style : 'padding-top:0.25em; text-align:left; font-style:90%;']
Comment Tag: 
                    
k: null
Text: ''''Notes''''
Inline Tag: br attributes: []
var: footnotes [null]
Close Tag: div
Text: ' '
End Template: #if:<footnotes:>
             
Text: '
'
End Template: Infobox

Comment Tag: 


Begin Template: #if:<author_abbreviation_zoo:<author_abbrev_zoo:>>
     
k: null
Begin Reference: Category:Zoologists with author abbreviations<!--
         -->
k: null
var: author_abbreviation_zoo [var: author_abbrev_zoo [null]]
End Reference: Category:Zoologists with author abbreviations<!--
         -->
Comment Tag: 

End Template: #if:<author_abbreviation_zoo:<author_abbrev_zoo:>>
     
Begin Template: #if:<pronunciation:>
k: null
Begin Reference: Category:Biography template using pronunciation
End Reference: Category:Biography template using pronunciation
End Template: #if:<pronunciation:>
Open Tag: noinclude attributes: []
Text: '
'
Begin Template: documentation
End Template: documentation
Text: '
'
Comment Tag: -Please add metadata to the  section at the bottom of the /doc subpage-
Text: '
'
Close Tag: noinclude
Text: '
'
End Document
